Void adds a custom runscript and udev rules to work around missing systemd.

--- a/install/lvm2
+++ b/install/lvm2
@@ -19,6 +19,14 @@
         'raid456' \
         'dm-integrity'
 
+    # from lvm2
+    map add_binary \
+        'lvm' \
+        'lvmetad'
+
+    # from device-mapper
+    add_binary 'dmsetup'
+
     # from thin-provisioning-tools
     add_binary 'pdata_tools'
     while IFS='' read -r -d '' symlink; do
@@ -39,18 +47,14 @@
         '11-dm-lvm.rules' \
         '13-dm-disk.rules' \
         '95-dm-notify.rules' \
-        '/usr/lib/initcpio/udev/11-dm-initramfs.rules'
-
-    # this udev rule is specific for systemd and non-systemd systems
-    if declare -F add_systemd_unit &>/dev/null; then
-        add_udev_rule '69-dm-lvm.rules'
-    else
-        add_udev_rule '/usr/lib/initcpio/udev/69-dm-lvm.rules'
-    fi
+        '/usr/lib/initcpio/udev/11-dm-initramfs.rules' \
+        '/usr/lib/initcpio/udev/69-dm-lvm-metad.rules'

     # config file
     sed -e 's/^[[:space:]#]*monitoring = [[:digit:]]\+\s*$/\tmonitoring = 0/' \
         -e '/^$/d' -e '/^[[:space:]]*#/d' "/etc/lvm/lvm.conf" | add_file - '/etc/lvm/lvm.conf' 644
+
+    add_runscript
 }

 help() {
